crates/pop-cli/src/commands/new/contract.rs

```diff
@@ -104,10 +100,14 @@ async fn guide_user_to_generate_contract() -> anyhow::Result<NewContractCommand>
 
 	let template = display_select_options(contract_type)?;
 
+	let name: String = input("Name of your contract?")
+		.placeholder("./my_contract")
```

Comment:
Note that this is not consistent with https://github.com/r0gue-io/pop-cli/blob/9644bb46b1582972315cc3987f87f274d794916a/crates/pop-cli/src/commands/new/parachain.rs#L126 which uses a hypen, which is the goal of this PR.

Reply:
You are right, changed. I'll keep the same you suggested  for consistency. The alternative was: `Where should your contract be created?`